src/action.cpp

```diff
@@ -536,16 +536,6 @@ std::string press_x( action_id act, const std::string &key_bound_pre,
     input_context ctxt = get_default_mode_input_context();
     return ctxt.press_x( action_ident( act ), key_bound_pre, key_bound_suf, key_unbound );
 }
-std::optional<std::string> press_x_if_bound( action_id act )
```

Comment:
function used once in src/help.cpp, deleting it. It'll be easier to refactor the rest of these `press_x` functions later on.